book/books_handler.rb

```diff
@@ -1,16 +1,19 @@
+require 'json'
 require_relative './book'
 class BooksHandler
-  attr_accessor :books
+  attr_accessor :books, :file
 
   def initialize
-    @books = []
+    @file = './jsonfiles/books.json'
```

Comment:
- [ ] Kindly make sure the missing file case is being handled, you can handle it by adding this line `return [] unless File.exists?("./jsonfiles/books.json")` and make sure to handle it for person and rental also.